### PR TITLE
Bump version to 3.0.0-SNAPSHOT and Kafka dependency to 0.10.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by ewencp**

### CODE REVIEW

**Comment by ewencp** [comment]
@miguno @ijuma @gwenshap @norwood Updating versions to clean up issues with system tests to address some issues in confluentinc/muckrake#90. Personally verified on that PR that it fixes issues w/ incompatible changes. These changes will be required for release anyway, modulo the SNAPSHOT suffix. Note that this upgrades all deps to 0.10.0.0-SNAPSHOT. Previous discussion left it unclear whether we might want to depend on older versions in some cases, but it simplifies things for now if we maintain one version across all packages.

Depends on confluentinc/schema-registry#298

Note that removal of some checks seemed necessary because checksums changed w/ addition of timestamps to messages. @Ishiihara would you mind also reviewing this since you've looked at this repo more closely? I'd like to keep the changes to a minimum while maintaining compatibility, if possible.

**Comment by ijuma** [comment]
Version changes look good, not sure about the checksum removal, will leave that to @Ishiihara

**Comment by ewencp** [comment]
@Ishiihara Updated the patch to still validate the checksum. I think the original code could probably have just been replaced entirely by a single call to `Message.computeChecksum` instead, but this seems like the minimal change that works with the new `Message` class.

**Comment by Ishiihara** [comment]
@ewencp Thanks for updating the patch. The change looks good to me.

File: camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/KafkaReader.java [hunk 1]
```diff
@@ -98,11 +99,13 @@ public boolean hasNext() throws IOException {
    * @return true if there exists more events
    * @throws IOException
    */
-  public boolean getNext(EtlKey key, BytesWritable payload, BytesWritable pKey) throws IOException {
+  public boolean getNext(EtlKey key, BytesWritable payload, BytesWritable pKey,
```
Comment (Ishiihara) [inline camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/KafkaReader.java:103]: Probably add messageRef to java doc.

**Pull request merged**